samples/nrf9160/modem_shell/src/shell.c

```diff
@@ -223,3 +227,9 @@ SHELL_CMD_REGISTER(rest, NULL,
 	"REST client.",
 	rest_shell);
 #endif
+
+#if defined(CONFIG_MOSH_CLOUD)
+SHELL_CMD_REGISTER(cloud, NULL,
+	"MQTT connection to nRFCloud.",
```

Comment:
We should have a discussion on the scope of "cloud" command. Whether it's intended just for nRF Cloud or whether it could be extended later. No changes required here I guess if we don't plan to make the command "nrf_cloud".

Reply:
agree. I have had thoughts for this command in a widers scope, i.e. beyond mqtt and nrf cloud. ...but maybe later.

Reply:
Some thoughts: if we plan to add other clouds, I would make the command something like "cloud connect amazon". In this case, the current naming would be ok I think. And if we don't add other clouds, then there is no point to change the name to nrf_cloud as there is no way to confuse it with any other cloud :)

Reply:
Yep, as a command name, `cloud` is most future-proof. And I think the current usage is fine. However, I reserve right to suggest backwards incompatible changes if we ever add more clouds behind this command :)
I mean mainly that `cloud connect` wouldn't necessarily be enough for nRF Cloud connection anymore then.